### PR TITLE
Tern unit

### PR DESCRIPTION
**Pull request by Lachlan-Charles**
Request to create unit:M3-PER-HA (Cubic Metre per Hectare)

### CODE REVIEW

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -12503,6 +12503,19 @@ unit:M3
   rdfs:label "Cubic Meter"@en-us ;
   rdfs:label "Cubic Metre"@en ;
 .
+unit:M3-PER-HA
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  qudt:conversionMultiplier 0.0001 ;
+  qudt:symbol "m^3/ha" ;
+  qudt:hasDimensionVector qkdv:A0E0L3I0M0H0T0D0 ;
```
Comment (jmkeil) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:12511]: `qkdv:A0E0L1I0M0H0T0D0` not `qkdv:A0E0L3I0M0H0T0D0`

**Review by steveraysteveray — APPROVED** [review]

**Comment by steveraysteveray** [comment]
@jmkeil, there is a trivial merge conflict, but I always get messed up when I try to resolve them from another repository. Would you mind merging qudt:master into your branch?

**Comment by jmkeil** [comment]
@steveraysteveray: There seems to have been a mix-up here. This PR is by @Lachlan-Charles.

**Comment by steveraysteveray** [comment]
Whoops! My mistake. @Lachlan-Charles, would you mind resolving the merge conflict? Thanks.

**Comment by Lachlan-Charles** [comment]
> 

Hi @steveraysteveray - sorry, could you please let me know how to resolve the merge conflict?  I have tried to to resolve it - but cant figure out how to do it.  Thanks

**Comment by steveraysteveray** [comment]
Take a look at https://phoenixnap.com/kb/how-to-resolve-merge-conflicts-in-git and see if you can resolve things. The only conflict is that you added a blank line after line 17075, which is harmless. But git isn't sure, so it is asking for you to intervene.

I'm assuming you have git installed on your machine. Give it a try, and I'll take a look after you do another push of your branch.

**Comment by Lachlan-Charles** [comment]
> Take a look at https://phoenixnap.com/kb/how-to-resolve-merge-conflicts-in-git and see if you can resolve things. The only conflict is that you added a blank line after line 17075, which is harmless. But git isn't sure, so it is asking for you to intervene.
> 
> I'm assuming you have git installed on your machine. Give it a try, and I'll take a look after you do another push of your branch.

Thanks Steve - I deleted the extra line (at 17089) and created a pull request - hope that fixes the issues.  Thanks

**Comment by steveraysteveray** [comment]
@Lachlan-Charles, I went ahead and resolved the merge conflict into a local branch and then merged that into master. I would have pushed the resolved branch back to your fork, but did not have permission to do so.
So it is all done now.

**Pull request merged**